### PR TITLE
Added new RHEL Upgrade Section

### PR DESCRIPTION
**Pull request by sabuchan07**
Cherry-pick into:

* [ x] Foreman 3.2
* [x ] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Comment by mdolezelova** [comment]
Hi @sabuchan07  thank you for submitting this PR. I've just noticed one small thing. I think this content is for Satellite 6.11, isn't it? If it is, you'd need to request cherry-picking to the branch 3.1 as well (as this one is the base for Satellite 6.11 docs). Or am I missing something? Thank you!

**Comment by sabuchan07** [comment]
@mdolezelova It IS for 6.11.  I submitted it yesterday before getting your email about the branches this morning.  It also doesn't pass the checks, so I need to do some troubleshooting and then I'll resubmit.  Thanks!

File: guides/doc-Upgrading_and_Updating/master.adoc [hunk 1]
```diff
@@ -74,6 +74,11 @@ ifdef::katello,orcharhino,satellite[]
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::katello,orcharhino,satellite[]
+//Migrating Linux 7 to Linux 8
+include::migrating_linux_7_to_linux_8.adoc[leveloffset=+2]
```
Comment (inoxx03) [inline guides/doc-Upgrading_and_Updating/master.adoc:79]: @sabuchan07 I think I found the issue
```suggestion
include::topics/migrating_linux_7_to_linux_8.adoc[leveloffset=+2]
```

Comment (inoxx03) [reply guides/doc-Upgrading_and_Updating/master.adoc:79]: I'd recommend that you try accepting the suggested change to the path. When you do this, you'll trigger a re-run of the automated checks, and with the path fixed and the include statement resolving correctly, your PR should pass all of them.

**Comment by maximiliankolb** [comment]
If you add new files, please put them directly into guides/common/modules; stay with only level1 headings, and exactly one level1 heading per file. You can use `[leveloffset=+1/2/3]` when including the file.

Please also add anchors and remove comments and unnecessary whitespace. Feel free to ping me/anyone for specific questions. I recommend you have a look at another file first to get a feel for our conventions etc, for example: https://github.com/theforeman/foreman-documentation/blame/master/guides/common/modules/proc_adding-custom-deb-repositories.adoc.

Thank you!

**Comment by evgeni** [comment]
I guess the changes here need to be coordinated with https://github.com/theforeman/foreman-documentation/pull/1248?

File: guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Perform an in-place migration on your server using the Leapp utility.
+
+[discrete]
+=== Fresh Install Method
+
+//.Unordered list
+
+* The fresh install migration method allows you to create a RHEL 8 system with identical specifications and the same hostname as the previous RHEL 7 system.
+
+* You can choose how much of the RHEL 7 system to include in the backup, allowing you to import only relevant parts of the RHEL 7 system to RHEL 8.
```
Comment (ehelms) [inline guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc:23]: I am unclear what this statement means for this method. @evgeni do you know?

File: guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+* This method requires a considerable amount of system downtime as the RHEL 8 system restores the system backup; to reduce downtime, you can use the RHEL 7 system on the target machine as the backup restores.
```
Comment (ehelms) [inline guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc:25]: Is this implying taking the backup results in considerable downtime? Or is this implying that users will take down their RHEL 7 based installation during the restore process thus resulting in downtime?

File: guides/doc-Upgrading_and_Updating/master.adoc [hunk 4]
```diff
@@ -74,6 +74,11 @@ ifdef::katello,orcharhino,satellite[]
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::katello,orcharhino,satellite[]
+//Migrating Linux 7 to Linux 8
+include::topics/migrating_linux_7_to_linux_8.adoc[leveloffset=+2]
```
Comment (maximiliankolb) [inline guides/doc-Upgrading_and_Updating/master.adoc:79]: Please move this file to modules.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+There are two ways to migrate your server to RHEL 8:
+
+//.Unordered list
+
+* Create a backup of your server on RHEL 7, perform a fresh install of a minimal RHEL 8 instance, and restore the backup on the RHEL 8 instance, or
+* Perform an in-place migration on your server using the Leapp utility.
+
+[discrete]
+=== Fresh Install Method
```
Comment (ehelms) [inline guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc:17]: Making a note that we will need a provision in this section for users with external databases.

**Pull request closed without merging**